client/my-sites/posts/post-list.jsx

```diff
@@ -277,7 +277,7 @@ var Posts = React.createClass( {
 				/>
 			);
 		} else {
-			if ( this.props.loading || ! this.props.sites.fetched ) {
+			if ( this.props.loading || ! this.props.haveSites ) {
```

Comment:
I wonder if we should just ditch the `sites` checks and add another FIXME? We shouldn't block this PR by [yours](https://github.com/Automattic/wp-calypso/pull/13094) which I guess would provide us with fetching state?